kyuubi-server/src/test/scala/org/apache/kyuubi/RestFrontendTestHelper.scala

```diff
@@ -50,7 +50,9 @@ object RestFrontendTestHelper {
 
 trait RestFrontendTestHelper extends WithKyuubiServer {
 
-  override protected lazy val conf: KyuubiConf = KyuubiConf()
+  override protected lazy val conf: KyuubiConf = {
+    KyuubiConf().set(KyuubiConf.SERVER_INFO_PROVIDER, "SERVER")
```

Comment:
overwrite the conf in `SessionsResourceSuite#test("get infoType")#SessionOpenRequest`